### ISSUE TITLE
Clean up syntax check page

### ISSUE THREAD

**Issue by larshp** [larshp/abapGit#1885]
Quite difficult to read, change to table instead?

![image](https://user-images.githubusercontent.com/5888506/44910787-ce843980-ad24-11e8-91b6-413cf7386661.png)

**Comment by patvo** [comment]
@larshp I really like the SCI result page. If you define a check variant like SYNTAX-ONLY, you have clickable links to jump in the code directly and you see a nice color code.
Just take this and apply it to the syntax check page. :wink: 

![2018-09-11 08_21_20-workspace - sc6_100_pvollet_de - eclipse platform](https://user-images.githubusercontent.com/35031904/45342027-f4b79e00-b59c-11e8-94bb-13924ab3c7b2.png)

**Comment by larshp** [comment]
yeah, perhaps that can be reused, however I'd like to keep this as a separate option, as it does not require any setup

**Comment by patvo** [comment]
Yes, I would like to see this being separate options in the future. I was just talking about the layout of the result page.

**Comment by marcellourbani** [comment]
What about using this and creating the variant on the fly?
Should be possible, as CL_CI_CHECKVARIANT is not final

**Comment by marcellourbani** [comment]
I think it's easy enough to reuse, having a go now

**Comment by marcellourbani** [comment]
Created a pull request for it
